apps/frontend/src/components/containers/Auth/Shop/ItemsShowcase/ItemsShowcase.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from 'react';
+import { AxiosError } from 'axios';
+import { useDispatch } from 'react-redux';
+
+import type { TItem } from '@/types/item';
+import useApi from '@/utils/useApi';
+import ItemsShowcaseView from './ItemsShowcase.view';
+
+const ItemsShowcase = () => {
+	const dispatch = useDispatch();
+
+	const [allItemsState, setAllItemsState] = useState<TItem[]>([]);
+
+	useEffect(() => {
+		const findAllItems = async () => {
+			try {
+				const response = await useApi(
+					{
+						url: `${import.meta.env.VITE_BACkEND_URL}/item/getallitems`,
```

Comment:
```suggestion
						url: `${import.meta.env.VITE_BACkEND_URL}/item/get-all-items`,
```

READ - https://restfulapi.net/resource-naming/